src/uct/api/uct.h

```diff
@@ -880,6 +880,22 @@ void uct_config_print(const void *config, FILE *stream, const char *title,
 
 /**
  * @ingroup UCT_CONTEXT
+ * @brief Get value by name from interface/MD configuration.
+ *
+ * @param [in]  config        Configuration to get from.
+ * @param [in]  name          Configuration variable name.
+ * @param [out] value         Pointer to get value. Should be allocated/freed by
+ *                            caller.
+ * @param [in]  max           Available memory space at @a value pointer.
+ *
+ * @return Error code.
+ */
+ucs_status_t uct_config_get(void *config, const char *name, char *value,
```

Comment:
In the error code how do we separate between wrong argument and the name is not found ?
It would be nice to include the debug capabilities that will help debug/trace the function.